components/org.wso2.carbon.identity.oauth.dcr/src/test/java/org/wso2/carbon/identity/oauth/dcr/service/DCRManagementServiceTest.java

```diff
@@ -257,7 +258,7 @@ public void registerOAuthApplicationWithNewSPWithRedirectUri(String oauthConsume
 
         when(mockRegistrationRequestProfile.getRedirectUris()).thenReturn(redirectUris);
 
-        mockOAuthAdminService = mock(OAuthAdminService.class);
+        OAuthAdminService mockOAuthAdminService = mock(OAuthAdminService.class);
```

Comment:
Can't we use an actual instance?

Reply:
I wanted to avoid actual method call **getOAuthApplicationDataByAppName** from OAuthAdminService. So that I have mocked it.